data/lang/module-findperson/en.json

```diff
@@ -0,0 +1,550 @@
+  },
+  "INTROTEXT_TRAITOR_FEMALE_1": {
+    "description": "",
+    "message": "Hello, my name is {client}. {wanted} didn't return from a holiday trip to her workplace at the {company}. She was last seen in the {system} ({sectorx}, {sectory}, {sectorz}) system. {wanted} has knowledge of some vital business secrets. So we need someone who can get a few answers from her about her ongoing absence. We pay {cash} for this mission. {system} is in a distance of {dist} ly."
+  },
+  "INTROTEXT_TRAITOR_FEMALE_2": {
+    "description": "",
+    "message": "Good day. The {company} is searching for an employee who stole some classified documents. Her name is {wanted}. Management calls her a traitor, but wants to give her the opportunity to reconsider and repent. We pay {cash} if you can find her in the {system} ({sectorx}, {sectory}, {sectorz}) system, {dist} ly distance, and deliver a message to her."
+  },
+  "INTROTEXT_TRAITOR_MALE_1": {
+    "description": "",
+    "message": "Hello, my name is {client}. {wanted} didn't return from a holiday trip to his workplace at the {company}. He was last seen in the {system} ({sectorx}, {sectory}, {sectorz}) system. {wanted} has knowledge of some vital business secrets. So we need someone who can get a few answers from him about his ongoing absence. We pay {cash} for this mission. {system} is in a distance of {dist} ly."
+  },
+  "INTROTEXT_TRAITOR_MALE_2": {
+    "description": "",
+    "message": "Good day. The {company} is searching for an employee who stole some classified documents. His name is {wanted}. Management calls him a traitor, but wants to give him the opportunity to reconsider and repent. We pay {cash} if you can find him in the {system} ({sectorx}, {sectory}, {sectorz}) system, {dist} ly distance, and deliver a message to him."
+  },
+  "IS_THERE_A_RISK": {
+    "description": "",
+    "message": "Is there any risk?"
+  },
+  "MERCENARY_GREETING_FEMALE_1": {
+    "description": "",
+    "message": "I'm sorry. Your death is my business!"
+  },
+  "MERCENARY_GREETING_FEMALE_2": {
+    "description": "",
+    "message": "I'll cut you to slices!"
+  },
+  "MERCENARY_GREETING_FEMALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? I will make you meet your maker first!"
+  },
+  "MERCENARY_GREETING_MALE_1": {
+    "description": "",
+    "message": "It's nothing personal. I kill you for a fee!"
+  },
+  "MERCENARY_GREETING_MALE_2": {
+    "description": "",
+    "message": "I'll send you to hell!"
+  },
+  "MERCENARY_GREETING_MALE_3": {
+    "description": "",
+    "message": "You want to meet {wanted}? Over my dead body!"
```

Comment:
How about "over _your_ dead body"?
More fun, and I think it works?

Reply:
How about both? `"MERCENARY_GREETING_MALE_4": {`

Reply:
Plausible. I err on many strings and little variation. :)